frontend/device/android/device.lua

```diff
@@ -9,6 +9,7 @@ local logger = require("logger")
 local util = require("util")
 local _ = require("gettext")
 local T = FFIUtil.template
+local realpath = FFIUtil.realpath
```

Comment:
Caching that in a local is not strictly necessary here?

(Harmless enough, though, just unusual).